frontend/src/components/stages/survey_editor_menu.scss

```diff
@@ -4,6 +4,7 @@
 
 :host {
   @include common.flex-column;
+  margin-top: -30px;
```

Comment:
Nit: Instead of doing this, can we just wrap the title and survey editor in a horizontal flex? (Otherwise, this creates a weird vertical flex layout because the survey editor menu gets squished.)

In survey_editor:
```
<div class="section">
  <div class="header">
    <div class="title">Survey questions</div>
    <survey-editor-menu></survey-editor-menu>
  </div>
  ...
</div>
```

Reply:
Wow, yes, done!